src/serialize.ts

```
@@ -39,17 +39,16 @@ const Serializer = /*@__PURE__*/ (function () {
       const typeB = typeof b;
 
       if (typeA === "string" && typeB === "string") {
-        return a.localeCompare(b);
+        return a < b ? -1 : a > b ? 1 : 0;
```

ESLint

[error] Nested ternary expression should be parenthesized.

(unicorn/no-nested-ternary)